stable/insights-admission/templates/configuration.yaml

```diff
@@ -8,6 +12,9 @@ metadata:
     {{- end }}
 webhooks:
 - admissionReviewVersions:
+{{- if .Capabilities.APIVersions.Has "admissionregistration.k8s.io/v1" }}
+  - v1
+{{- end }}
   - v1beta1
```

Comment:
should we keep this one or put it inside an `else`?

Reply:
> should we keep this one or put it inside an    else   ?

I **believe** we want both, but am not 100%. I'm not clear on who/what may send what versions of AdmissionReview objects to our webhook. 

I looked at [this doc](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/) when creating this PR - it says this about that field:
Webhooks can specify what versions of    AdmissionReview objects they accept with the admissionReviewVersions    field in their configuration ...
Webhooks are required to support at least one AdmissionReview version understood by the current and previous API server.
API servers send the first AdmissionReview version in the admissionReviewVersions list they support. If none of the versions in the list are supported by the API server, the configuration will not be allowed to be created. If an API server encounters a webhook configuration that was previously created and does not support any of the AdmissionReview versions the API server knows how to send, attempts to call to the webhook will fail and be subject to the failure policy.

@rbren / @makoscafee  What is your interpretation of the above, and whether we want both fields?

Reply:
I would go with leaving both fields there... I am assuming from version `1.16` to `1.21` there are both capabilities `v1beta1` and `v1`

Reply:
Correct RE: 1.16-1.21.